src/Sulu/Bundle/ResourceBundle/Filter/Exception/FilterNotFoundException.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Bundle\ResourceBundle\Filter\Exception;
+
+/**
+ * Class FilterNotFoundException
+ */
+class FilterNotFoundException extends FilterException
+{
+    /**
+     * The name of the object not found
+     * @var string
+     */
+    private $entityName;
+
+    /**
+     * The id of the object not found
+     * @var integer
+     */
+    private $id;
+
+    public function __construct($id)
+    {
+        $this->entityName = 'SuluResourceBundle:Filter';
+        $this->id = $id;
+        parent::__construct('The filter with the id "'.$this->id.'" was not found.', 0);
```

Comment:
before and after the `.` there should be a blank.

```php
parent::__construct('The filter with the id "' . $this->id . '" was not found.', 0);
```

and i would use `sprintf` instead of string concatination!